src/app/hooks/useTransactions.ts

```diff
@@ -17,16 +18,38 @@ export const useTransactions = () => {
           accountId,
           limit,
         });
+        const fetchInvoice = await api.getInvoices({ isSettled: true, limit });
```

Comment:
the naming is wrong for this variable. Maybe `getInvoicesResponse`?

Reply:
resolved